seirmo/models.py

```diff
@@ -82,4 +79,10 @@ def simulate(self, parameters, times):
         sol = solve_ivp(lambda t, y: self._right_hand_side(t, y, c),
                         [times[0], times[-1]], y_init, t_eval=times)
 
-        return sol['y'].transpose()
+        if return_incidence is False:
+            return sol['y'].transpose()
+        elif return_incidence is True:
+            # Return incitance = (I(t_final) + R(t_final))
+            #   - (I(t_initial) + R(t_initial))
+            return (sol['y'][2, -1] + sol['y'][3, -1]) - (
```

Comment:
Is this really computed the incidence number in all time intervals? See above comment for another suggestion. Let me know whether all this numpy *magic* is confusing.

Reply:
Yeah sorry. I have fully understood now.